splunk_event/conf.yaml.example

```diff
@@ -48,9 +48,36 @@ init_config:
 # It is possible to specify multiple saved_searches on a single instance.
 instances:
   - url: "https://localhost:8089"
+
+    # username: "admin" # deprecated
+    # password: "admin" # deprecated
+
     # verify_ssl_certificate: false
-    username: "admin"
-    password: "admin"
+
+    ## new authentication structure either token or basic auth will be used.
+    authentication:
+      basic_auth:
+        username: "admin"
+        password: "admin"
+      # token_auth:
+        ## Token for the user who will be using it
+        # name: "api-user"
+
+        ## it's first initial valid token which will be exchanged with new generated token as soon as the check starts
+        ## first time and in case of restart, this token will not be used anymore
+        # initial_token: "my-initial-token-hash"
+
+        ## JWT audience used for purpose of token
+        # audience: "search"
+
+        ## token_expiration_days also works only in token based authentication and this value used to set expiry days for
+        ## create a new token after first initial token is provided. By default the value is 90 days. The token is renewed
```

Comment:
Suggestion: "create a new token after first initial token is provided". -> "When a token is about to expire, a new token is requested from Splunk. The validity of the newly requested token is requested to be `token_expiration_days` days. After `renewal_days` days the token will be renewed for another token_expiration_days days."